lnbits/static/js/event-reactions.js

```diff
@@ -1,16 +1,30 @@
-function eventReactionWebocket(event_id, event_function) {
+function eventReactionWebocket(event_id) {
   localUrl = ''
+  reaction = localStorage.getItem('lnbits.reactions')
+  if (!reaction || reaction === 'None') {
+    return
+  }
   if (location.protocol !== 'http:') {
     localUrl = 'wss://' + location.host + '/api/v1/ws/' + event_id
   } else {
     localUrl = 'ws://' + location.host + '/api/v1/ws/' + event_id
   }
   connection = new WebSocket(localUrl)
   connection.onmessage = function (e) {
-    event_function()
+    let parsedData = JSON.parse(e.data)
+    if (parsedData.payment.amount < 0) {
+      return
+    } else {
+      try {
+        reaction = localStorage.getItem('lnbits.reactions')
+        window[reaction.split('|')[1]]()
+      } catch (e) {
+        console.log(e)
+      }
+    }
```

Comment:
```suggestion
  connection.onmessage = function (e) {
    try {
      const parsedData = JSON.parse(e.data)
      if (parsedData.payment.amount < 0) {
        return
      }
      reaction = localStorage.getItem('lnbits.reactions')
      if (reaction) {
        window[reaction.split('|')[1]]()
      }
    } catch (e) {
      console.log(e)
    }
  }
```

- `data` might not be valid json
- `reactions` can be `null`